### PR TITLE
added "The fire triangle: how to mix substitution, dependent eliminat…

### PR DESCRIPTION
**Pull request by marklemay**
…ion, and effects"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,6 +196,10 @@ in various programming languages.
   by Georgios Karachalias, Matija Pretnar, Amr Hany Saleh, Stien Vanderhallen and Tom Schrijvers  
   ([arxiv](https://arxiv.org/abs/2005.13814))  
 
+* **The fire triangle: how to mix substitution, dependent elimination, and effects** (POPL 2020)
```
Comment (dhil) [inline README.md:199]: Please add two trailing whitespaces to both the title and author lines.

File: README.md [hunk 2]
```diff
@@ -196,6 +196,10 @@ in various programming languages.
...
+  by Pierre-Marie Pédrot and Nicolas Tabareau
```
Comment (dhil) [inline README.md:200]:
```suggestion
  by Pierre-Marie Pédrot and Nicolas Tabareau  
```

**Comment by marklemay** [comment]
what are the trailing white spaces for?

**Comment by dbuenzli** [comment]
They ensure the line break gets respected (otherwise the thing is treated as a paragraph to typeset).

**Pull request merged**